Makefile

```
@@ -80,21 +80,21 @@ docker-build:
 	docker build -t pydo:dev .
 
 .PHONY: docker-python
-docker-python: docker-build  ## Runs a python shel within a docker container
+docker-python: docker-build  ## Runs a python shell within a docker container
 	docker run -it --rm --name pydo pydo:dev python
 
 .PHONY: lint-docs
 lint-docs:
 	docker run -v $(ROOT_DIR):/workdir ghcr.io/igorshubovych/markdownlint-cli:latest "*.md"
 
 .PHONY: generate-docs
-generate-docs: install ## Generate documentation for Client using Sphinx
+generate-docs: install ## readthedocs requires a requirements.txt file, this step converts poetry file to requirements.txt file before re-gen the docs
 	@echo Generating documentation...;
 	@echo Converting poetry file to requirements.txt...; \
 	poetry export -f requirements.txt -o requirements.txt --without-hashes && \
```

This target is failing with:

```
| Generating documentation...
| Converting poetry file to requirements.txt...
| /bin/sh: 2:  : not found
| make: *** [Makefile:93: generate-docs] Error 127
```

Splitting some of these to separate commands resolves this:
```suggestion
	@echo Generating documentation...;
	@echo Converting poetry file to requirements.txt...; 
	poetry export -f requirements.txt -o requirements.txt --without-hashes
```